### ISSUE TITLE
Add auto resolver for firefox in popup

### ISSUE THREAD

**Issue by PatrykKuniczak** [Jonghakseo/chrome-extension-boilerplate-react-vite#641]
**Is your feature request related to a problem? Please describe.**
Add automatic solving of the path, depend of the selected script, like in other places

**Describe the solution you'd like**
Add 
```ts
files: isFirefox ? ['../content-runtime/index.iife.js'] : ['content-runtime/index.iife.js'],
```
to ```chrome.scripting.executeScript``` in ```popup.tsx```

and remove 
```
 /**
       * If you are using Firefox, you should use a relative path. :(
       * @example
       * files: ['../content-runtime/index.iife.js'],
       */
```

**Additional context**
i was trying to do it, but i can't import with ```process``` or ```import.meta``` this ```__FIREFOX__``` value.
@Jonghakseo if you have solution for that, it will be good to implement it

**Comment by govza** [comment]
files: ['/content-runtime/index.iife.js'] fixes the issue?

**Comment by PatrykKuniczak** [comment]
> `files: ['/content-runtime/index.iife.js']` fixes the issue for both browsers?

Yeah, it works indeed, good job 😄

**Comment by PatrykKuniczak** [comment]
@govza Let's change it in your #616 PR, because this is FF related, and it's not necessary to create other PR for this IMHO 😸

**Comment by govza** [comment]
Ok, let's add some more content fixes than.

**Comment by PatrykKuniczak** [comment]
closed by #616
